clients/java/src/main/java/io/camunda/zeebe/client/impl/ZeebeClientCloudBuilderImpl.java

```diff
@@ -177,6 +191,10 @@ private CredentialsProvider determineCredentialsProvider() {
       ensureNotNull("client id", clientId);
       ensureNotNull("client secret", clientSecret);
       final OAuthCredentialsProviderBuilder builder = new OAuthCredentialsProviderBuilder();
+
+      if (innerBuilder.isPlaintextConnectionEnabled()) {
+        Loggers.LOGGER.debug("Expected setting 'usePlaintext' to be 'false', but found 'true'.");
```

Comment:
A plain text connection won't work. What's the current behavior if the user specifies it? Should we overwrite it and always use a secure connection for the cloud builder? We would still log this, but mention that we're overwriting this.

This is a little bit out of scope, but the new log brought it up in my mind.

Reply:
I was following the other examples of log output in the class. Basically, we always print out a hint when an overwrite happens. 

I wouldn't use a hard override here, because it might be still interesting for people to create clients with configurations that won't work - e.g. in tests. 

So the log message makes it a little better, and I didn't want to change the behavior too much.